### PR TITLE
Fix appveyor compilation for python 3.4 and 3.5

### PR DESCRIPTION
**Pull request by vallsv**

### CODE REVIEW

File: ci/appveyor.yml [hunk 1]
```diff
@@ -32,48 +32,67 @@ environment:
...
-        # Python 3.4: disabled for now, investigate why it crashes
-        #- PYTHON_DIR: "C:\\Python34-x64"
-        #  PYTHON_VERSION: "3.4.3"
-        #  PYTHON_ARCH: "64"
-        #  DISTUTILS_USE_SDK: "1"
+        # Python 3.4
+        - PYTHON_DIR: "C:\\Python34-x64"
+          PYTHON_VERSION: "3.4.3"
+          PYTHON_ARCH: "64"
+          DISTUTILS_USE_SDK: "1"
```
Comment (t20100) [inline ci/appveyor.yml]: DISTUTILS_USE_SDK seems unused

**Comment by t20100** [comment]
DISTUTILS_USE_SDK variable seems useless.
Otherwise, it looks good to me.

**Comment by vallsv** [comment]
We fixed that with Jerome from the appveyor doc https://packaging.python.org/appveyor/
Then i thought it was accessed by the python library from the env.

But thanks for the warn. I think in our case it is really useless.
Let me rework a little the patch using this appveyor doc
cause the MSVC SetEnv.cmd must be called in difference cases: Python 3.3 x64 or Python 3.4 x64 only

**Comment by vallsv** [comment]
Ok, this one should be the right one. If appveyor is agree

**Pull request merged**